karma.conf.js

```diff
@@ -47,7 +47,7 @@ module.exports = function(config) {
     logLevel: config.LOG_INFO,
     autoWatch: true,
     browsers: [ 'ChromeCI' ],
-    singleRun: true,
```

Comment:
Please revert this, so it doesn't affect other, standard is to have single run :).